packages/stark-build/config/tslint.json

```diff
@@ -3,7 +3,7 @@
   "defaultSeverity": "error",
   "rules": {
     // Codelyzer recommended rules
-    "directive-selector": [true, "attribute", "", "kebab-case"],
+    "directive-selector": [true, "attribute", "", "camelCase"],
```

Comment:
I know angular material also does this, so it's fine for me. But do you happen to know the reason? 🤔

Reply:
Ah yes, well in fact this is mentioned in the Angular Style Guide ([style-02-06](https://angular.io/guide/styleguide#style-02-06) and [style-02-08](https://angular.io/guide/styleguide#style-02-08)) and it is also the [recommended codelyzer configuration](https://github.com/mgechev/codelyzer/tree/4.5.0#recommended-configuration) 😉.

Moreover, we already follow those guidelines so I didn't have to fix anything in our code 👍